cloud-formation/d2dbastion/stack.yml

```diff
@@ -0,0 +1,21 @@
+stack:
```

Comment:
Is this your own proprietary yml syntax? Will people have to learn this to use these scripts?